overlord/ifacestate/helpers.go

```diff
@@ -138,13 +156,6 @@ func (m *InterfaceManager) regenerateAllSecurityProfiles() error {
 
 		// For each backend:
 		for _, backend := range securityBackends {
-			// The issue this is attempting to fix is only
-			// affecting seccomp/apparmor so limit the work just to
-			// this backend.
-			shouldRefresh := (backend.Name() == interfaces.SecuritySecComp || backend.Name() == interfaces.SecurityAppArmor)
-			if !shouldRefresh {
-				continue
-			}
```

Comment:
This seems fine and is good for future-proofing. Since we are only doing this on refresh/revert of snapd (and different kernel capabilities), this extra work will not usually be felt, and the future-proofing benefits IMHO far outweigh the infrequently felt small optimization.